CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.1.1 (2019-03-27)
+
+- Fixed radio group component to support boolean values. [#30](https://github.com/blackbaud/skyux-forms/pull/30)
```

Comment:
Shouldn't this link to itself? #31

Reply:
Yep! I was trying to make a guess for the correct number before the PR was created. Thanks.